### DISCUSSION TITLE
MS-defender blocking vcpkg*.exe

### DISCUSSION THREAD

**Discussion by gvanem** [microsoft/vcpkg#27015] [General]
I just got a pop-up window from *Microsoft Defender* showing a Trojan in this
downloaded `vcpkg.exe` in my `%TEMP%\vcpkg` folder:

![Defender](https://user-images.githubusercontent.com/945271/192704918-f5ea1742-ce46-4b43-ba1e-a97f48a3cc98.png)

Probably a false positive, but what are the files in `%TEMP%\vcpkg` good for?

**Comment by Thomas1664** [comment]
This is an instance of vcpkg that was copied to `Temp` to upload metrics after vcpkg has finished its work. You can find more information on metrics in the [docs](https://github.com/microsoft/vcpkg#telemetry).
